libs/core/src/lib/feed-input/directives/feed-input-avatar.directive.ts

```diff
@@ -0,0 +1,45 @@
+import { Directive, ElementRef, Input, OnChanges, OnInit } from '@angular/core';
+import { applyCssClass } from '@fundamental-ngx/core';
```

Comment:
Could you add direct import there? Otherwise travis CI is going to fail 
![image](https://user-images.githubusercontent.com/26483208/99381969-183ca880-28cc-11eb-9d40-26fb6f42a3c2.png)